index.html

```diff
@@ -1,12 +1,12 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" prefix="og: https://ogp.me/ns/website#">
```

Comment:
Won't this affect other website resources? K didnt findbany info that it can break something, but it worth to double check in different social networks